exporters/otlp/otlp_test.go

```diff
@@ -338,3 +343,72 @@ func TestSplitDriverFail(t *testing.T) {
 		}
 	}
 }
+
+func ExampleNewExporter() {
```

Comment:
These example tests can be moved to their own `example_test.go` file so they do not get lost in the unit tests here.